games/static/games/proto.js

```diff
@@ -139,8 +139,11 @@ class Board {
         this.alienHealthLevels = Object.values(this.setupObject)
         this.direction = 1
         this.defenderLocation = this.defenderStartPosition()
+        this.Left = -1
```

Comment:
find out how to convert this to const
when u save something as a class member it means you can override it
consts can be static variables or maybe even outside the class